e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js

```diff
@@ -535,6 +536,33 @@ describeEE("scenarios > embedding > full app", () => {
         ).to.be.true;
       });
     });
+
+    it("should allow downloading question results when logged in via Google SSO (metabase#39848)", () => {
+      const CSRF_TOKEN = "abcdefgh";
+      cy.intercept("GET", "/api/user/current", req => {
+        req.on("response", res => {
+          res.headers["X-Metabase-Anti-CSRF-Token"] = CSRF_TOKEN;
```

Comment:
I just mock the header as it's virtually impossible to set up Google SSO at the moment.